lib/salus/scanners/language_version/ruby_version_scanner.rb

```diff
@@ -0,0 +1,37 @@
+require 'salus/scanners/language_version/base'
+
+module Salus::Scanners::LanguageVersion
+  class RubyVersionScanner < Base
+    def self.supported_languages
+      %w[ruby]
+    end
+
+    private
+
+    def run_version_scan?
+      ruby_project?
+    end
+
+    def lang_version
+      @lang_version ||= ruby_project? ? ruby_version : nil
+    end
+
+    # '.ruby-version' file stores the ruby version used in the ruby project
+    # Following is an example for running 'cat .ruby-version':
+    # 2.7.1
+    def ruby_version
+      shell_return = Dir.chdir(@repository.path_to_repo) do
```

Comment:
I'd like to honor the Gemfile / Gemfile.lock too as fall backs

Reply:
We should be able to pull the ruby version out of there too.

So desired precedence 

.ruby-version
Gemfile.lock
Gemfile

Reply:
Updated.